src/main/java/org/zaproxy/zap/extension/neonmarker/ExtensionNeonmarker.java

```diff
@@ -62,33 +62,34 @@
     private NeonmarkerPanel neonmarkerPanel;
     private MarkItemColorHighlighter highlighter = null;
 
-    static Color[] palette = {
-        // RAINBOW HACKER THEME
-        new Color(0xff8080),
-        new Color(0xffc080),
-        new Color(0xffff80),
-        new Color(0xc0ff80),
-        new Color(0x80ff80),
-        new Color(0x80ffc0),
-        new Color(0x80ffff),
-        new Color(0x80c0ff),
-        new Color(0x8080ff),
-        new Color(0xc080ff),
-        new Color(0xff80ff),
-        new Color(0xff80c0),
-        // CORPORATE EDITION
-        new Color(0xe0ffff),
-        new Color(0xa8c0c0),
-        new Color(0x708080),
-        new Color(0x384040),
-        // Placeholder
-        PLACEHOLDER
-    };
+    static List<Color> palette =
+            Arrays.asList(
```

Comment:
This returns a fixed size list, attempting to add more elements will lead to an exception. You need to use other implementation (i.e. pass it into an `ArrayList`).

Reply:
Tweaked, let me know what you think.

Reply:
Looks good.

Reply:
Thanks!

Reply:
No problem.